assets/dev/js/frontend/frontend.js

```diff
@@ -294,6 +294,16 @@ export default class Frontend extends elementorModules.ViewModule {
 		jQuery.migrateTrace = false;
 	}
 
+	populateActiveBreakpointsConfig() {
+		this.config.responsive.activeBreakpoints = {};
+
+		Object.keys( this.config.responsive.breakpoints ).forEach( ( breakpointKey ) => {
```

Comment:
```suggestion
		Object.keys( this.config.responsive.breakpoints ).forEach( ( breakpointKey, breakpointData ) => {
```

Reply:
It's `object.keys`. I changed it to `object.entries` and made the changes according to your comments. It works.